packages/amplify-category-api/src/index.ts

```diff
@@ -34,7 +34,6 @@ export {
   generateContainersArtifacts,
   processDockerConfig,
 } from './provider-utils/awscloudformation/utils/containers-artifacts';
-export { getAuthConfig } from './provider-utils/awscloudformation/utils/get-appsync-auth-config';
```

Comment:
This export is needed in `packages/amplify-provider-awscloudformation/src/graphql-transformer/transform-graphql-schema.ts`.

Reply:
Thanks, I overlooked that.